lib/renderer.coffee

```diff
@@ -143,14 +135,19 @@ tokenizeCodeBlocks = (html, defaultLanguage='text') ->
       if fenceName is defaultLanguage
         preElement.className = ''
       else
-        highlighter ?= new Highlights(registry: atom.grammars)
-        highlightedHtml = highlighter.highlightSync
+        highlightedHtml = highlight
           fileContents: codeBlock.text()
           scopeName: scopeForFenceName(fenceName)
+          nbsp: true
```

Comment:
I don't think we need the leading `&nbsp;` characters. That just adds a lot of junk to the generated HTML. I tested this by setting it to false and got the same result. I think it's safe to disable it (it's enabled by default).